homework6/baseballscripts.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE historicalAverage(
```

Comment:
I do not know how this sql is automatically executed, not included in the bash file.